### PR TITLE
Remove images borders, scale wide images a bit larger

### PR DESCRIPTION
**Pull request by nathanstitt**
image

### CODE REVIEW

File: tutor/src/components/book-page.js [hunk 1]
```diff
@@ -24,14 +24,33 @@ const LEARNING_OBJECTIVE_SELECTORS = '.learning-objectives, [data-type=abstract]
...
     || img.closest('[data-type=media]');
   if (!figure) { return; }
+
+  if (figure.querySelector('.splash')) {
+    figure.classList.add('full-width');
+  }
   if (figure.classList.contains('splash')) { return; }
+  // figures that are not in a series
+  if (!hasSiblingFigure(figure)) {
+    figure.classList.add('standalone');
```
Comment (Dantemss) [inline tutor/src/components/book-page.js]: Didn't you already do this on another PR but called this 'independent'?

Comment (Dantemss) [reply tutor/src/components/book-page.js]: Ah nvm I see you just renamed it

Comment (nathanstitt) [reply tutor/src/components/book-page.js]: yeah, i thought "standalone" was clearer but 🤷‍♂
